proc/threads.c

```diff
@@ -1601,12 +1613,17 @@ int proc_lockSetInterruptible(lock_t *lock)
 
 static int _proc_lockUnlock(lock_t *lock)
 {
-	thread_t *owner = lock->owner;
+	thread_t *owner = lock->owner, *current;
 	spinlock_ctx_t sc;
 	int ret = 0, lockPriority;
 
 	hal_spinlockSet(&threads_common.spinlock, &sc);
 
+	current = _proc_current();
+	(void)current; /* Unused in non-debug build */
```

Comment:
I'd wrap L1622 in an if macro so that it's present only when NDEBUG is set

Reply:
I'd rather avoid introducing `#if`s if possible